lib/ansible/module_utils/facts/system/distribution.py

```diff
@@ -464,6 +464,10 @@ def parse_distribution_file_CentOS(self, name, data, path, collected_facts):
             centos_facts['distribution_release'] = 'Stream'
             return True, centos_facts
 
+        if "TencentOS Server" in data:
```

Comment:
Thanks for the PR. Could you please add a test case for this change by using https://github.com/ansible/ansible/blob/devel/hacking/tests/gen_distribution_version_testcase.py (Like https://github.com/ansible/ansible/tree/devel/test/units/module_utils/facts/system/distribution/fixtures)?

Reply:
Yes I will take care of this tomorrow

Reply:
ok done.